server.py

```diff
@@ -361,6 +361,8 @@ def post(self, region):
 
         type = args['type']
         data = args['data']
+        # TODO add variable to get the excluded_phases for smashGG brackets
+        excluded_phases = args['excluded_phases']
```

Comment:
you need to add "excluded_phases" as an argument to the parser (lines 348-350) or this will throw an exception (this causes a bunch of tests to fail right now)